src/qibolab/instruments/qblox/cluster_qcm_bb.py

```diff
@@ -110,26 +110,28 @@ class ClusterQCM_BB(Instrument):
     )
 
     def __init__(self, name: str, address: str, cluster: Cluster = None):
-        """Initialises the instance.
+        """
+        Initialize a Qblox QCM baseband module.
 
-        All class attributes are defined and initialised.
+        Parameters:
+        - name: An arbitrary name to identify the module.
+        - address: The network address of the instrument, specified as "cluster_IP:module_slot_idx".
+        - cluster: The Cluster object to which the QCM baseband module is connected.
 
-        Args:
-            name(str): A unique name given to the instrument.
-            address: IP_address:module_number (the IP address of the cluster and module number)
-            cluster: the cluster to which the instrument belongs.
+        Example:
+        To create a ClusterQCM_BB instance named 'qcm_bb' connected to slot 2 of a Cluster at address '192.168.0.100':
+        >>> cluster_instance = Cluster("cluster","192.168.1.100", settings)
+        >>> qcm_module = ClusterQCM_BB(name="qcm_bb", address="192.168.1.100:2", cluster=cluster_instance)
         """
         super().__init__(name, address)
         self.ports: dict = {}
         self.settings: dict = {}
         self.device = None
-        self.channels: list = []
```

Comment:
I managed to delete the attributes `self.channels` and `self._port_channel_map`. 
However the attribute `self.channel_port_map` was kept since it is used in function `get_if()` that has no direct access to qubit objects.